src/Cli/dotnet/commands/VirtualProjectBuildingCommand.cs

```diff
@@ -0,0 +1,193 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+#nullable enable
```

Comment:
Is this needed? The other files seem to implicitly have nullable analysis enabled

Reply:
We're in a weird state with nullability in this codebase - definitely do what you need to do in order to enable the checks on the files you're working on. We can figure the rest out later.